entities/tokens.go

```diff
@@ -0,0 +1,20 @@
+package entities
+
+import (
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type TokenField string
+
+const (
+	TokenID      TokenField = "_id"
+	TokenJWT     TokenField = "jwt"
+	TokenCreator TokenField = "creator"
+)
+
+// Token is the struct to store tokens
+type Token struct {
+	ID      primitive.ObjectID `json:"_id" bson:"_id"`
+	JWT     string             `json:"jwt"  bson:"jwt" validate:"required"`
+	Creator primitive.ObjectID `json:"creator"  bson:"creator" validate:"required"`
```

Comment:
We'd want this information since in the frontend, we only want to display the service tokens to the user that created them. I don't think we'd want to display all the service tokens to all the users with the ability to create them.

I know initially the service tokens won't have a frontend, it will just be a manually invoked API, but I thought it was worthwhile to include it here

Reply:
Right, would we need to limit the `CreateServiceToken` to only be usable with a user token? With the current implementation of `ApiV2Router.CreateServiceToken`, we'd get an `ErrInvalidTokenType` from `Authorizer.GetUserIdFromToken` if the token is a service token.

Reply:
Yes, I don't think it should be allowed to generate service tokens using other service tokens. I can't really think of a use-case for that functionality.